src/SIL.LCModel.Core/SpellChecking/SpellEngine.cs

```diff
@@ -203,14 +261,19 @@ public void Dispose()
 
 		protected virtual void Dispose(bool disposing)
 		{
-			Debug.WriteLineIf(!disposing, "****** Missing Dispose() call for " + GetType().Name + ". ****** ");
+			#if __MonoCS__
 			if (m_hunspellHandle != IntPtr.Zero)
 			{
 				Hunspell_uninitialize(m_hunspellHandle);
 				m_hunspellHandle = IntPtr.Zero;
 			}
+			#else
+			Debug.WriteLineIf(!disposing, "****** Missing Dispose() call for " + GetType().Name + ". ****** ");
```

Comment:
this `WriteLineIf` should always be the first line of `virtual void Dispose(bool)`